hack/build.sh

```diff
@@ -78,7 +78,7 @@ for dir in ${dirs}; do
 
   IMAGE_NAME="${NAMESPACE}${BASE_IMAGE_NAME}-${dir//./}-${OS}"
 
-  if [[ -v TEST_MODE ]]; then
+  if [[ "${TEST_MODE}" == "true" ]]; then
```

Comment:
Right, It might be better to just leave this as it is and change the Makefile rules to not define TEST_MODE when not running `make test`. Thanks